oneview_redfish_toolkit/tests/test_util.py

```diff
@@ -254,9 +255,13 @@ def test_get_ip(self):
             self.fail("Failed to get a valid IP Address")
 
     @mock.patch.object(util, 'OneViewClient')
```

Comment:
Is this OneViewClient mock still required?

Reply:
No, once load_config() does not instantiate it anymore. Removing it.